tests/test_utils.py

```diff
@@ -52,3 +52,27 @@ def test_mpp2objective_power(_sample_svs):
 
     with pytest.raises(ValueError):
         utils.misc.mpp2objective_power(mpp=10)
+
+
+def test_contrast_enhancer():
+    """"Test contrast enhancement funcitionality."""
+    input_array = np.array(
```

Comment:
Can you elaborate in comments how you chose the values to run the tests?

Reply:
The goal is to check if `contrast_enhancer` method can reproduce results for uint8 images with 3 channels. Qualitative evaluation of the method has been done several times using example images. To check the performance and reproducibility in a quantitative way, and also to reduce uploading more images to the repository, a random tensor is generated and fed into the method. The `output_array ` should be exactly same as expected `result_array` (which has been numerically validated offline).

Reply:
Please can you add a short description as comments in the test itself? So anyone modifying or writing the code may understand it easily.

Reply:
yes sure.

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/cf87a50d92e62ea2597f9c4da77f9dcf2cc2491e